src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -71,7 +72,9 @@ const Body = BaseForm.extend({
     const checkPort = () => {
       return request({
         url: getAuthenticatorUrl('probe'),
-        timeout: 1000 // if authenticator & its probing endpoint exist, it should respond within 1000ms
+        // if authenticator & its probing endpoint exist, it should respond within 1000ms
```

Comment:
IMO, the comment should gives context of what the change is for and any pending TODOs. In our case, port checking in Android is slower because that SSL handshake is needed, without increasing timeout, request may fail sometimes. While other platform (Windows) uses http only at the moment, response time is within 1000 millseconds. So once Windows switches to use https, we will face the same issue. At the same time, we need to investigate and see how to improve the response time.